website/docs/openmp.md

```diff
@@ -0,0 +1,34 @@
+Enable or disable [OpenMP](https://en.wikipedia.org/wiki/OpenMP).
+
+```lua
+openmp "value"
+```
+If no value is set for a configuration, the toolset's default OpenMP option (usually Off) will be performed.
+
+### Parameters ###
+
+`value` is one of:
+
+| Value   | Description                                       |
+|---------|---------------------------------------------------|
+| On      | Turn on OpenMP.                                   |
+| Off     | Turn off OpenMP.                                  |
+
+### Applies To ###
+
+Project configurations.
+
+### Availability ###
+
+Require a version 5.0 beta 1 or later. Currently only implemented for Visual Studio 2010+. As a workaround for other toolsets, you can use [buildoptions](buildoptions.md) like this:
```

Comment:
So close! The only nitpick I can find is the phrasing here is a little confusing and doesn't match the rest of the documentation. Instead of "Require a version 5.0 beta 1 or later" make it "Premake 5.0-beta1 or later.". See [filter](https://premake.github.io/docs/filter/#availability) for an example. Thanks so much for putting this together and dealing with all of our change requests! :)